lib/fastlane/plugin/ddg_apple_automation/actions/asana_log_message_action.rb

```diff
@@ -0,0 +1,96 @@
+        is_scheduled_release = params[:is_scheduled_release] || false
+        github_handle = params[:github_handle]
+
+        automation_subtask_id = AsanaGetReleaseAutomationSubtaskIdAction.run(task_url: task_url, asana_access_token: token)
+
+        if is_scheduled_release
+          task_id = AsanaExtractTaskIdAction.run(task_url: task_url)
+          assignee_id = AsanaExtractTaskAssigneeAction.run(task_id: task_id, asana_access_token: token)
+        else
+          if github_handle.to_s.empty?
+            UI.user_error!("Github handle cannot be empty for manual release")
+            return
+          end
+          assignee_id = AsanaGetUserIdForGithubHandleAction.run(github_handle: github_handle, asana_access_token: token)
+        end
+
+        asana_client = Asana::Client.new do |c|
+          c.authentication(:access_token, token)
+        end
+
+        begin
+          asana_client.tasks.add_followers_for_task(task_gid: automation_subtask_id, followers: [assignee_id])
+        rescue StandardError => e
+          UI.user_error!("Failed to add a collaborator to the release task: #{e}")
+        end
+
+        AsanaAddCommentAction.run(task_id: automation_subtask_id, comment: comment, template_name: template_name, asana_access_token: token)
+      end
+
+      def self.description
+        "Add a Message to Asana Release Automation Task"
+      end
+
+      def self.authors
+        ["DuckDuckGo"]
+      end
+
+      def self.return_value
+        ""
+      end
+
+      def self.details
+        "Adds a comment about release progress to the Asana release task's 'Automation' subtask"
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.asana_access_token,
+          FastlaneCore::ConfigItem.new(key: :task_url,
+                                       description: "Asana release task URL",
+                                       optional: false,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :comment,
+                                       description: "Comment to add to the Asana task",
+                                       optional: true,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :template_name,
+                                       description: "Name of a template file (without extension) for the comment. Templates can be found in assets/asana_add_comment/templates subdirectory.
+      The file is processed before being sent to Asana",
+                                       optional: true,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :github_handle,
+                                       description: "Github user handle",
+                                       optional: true,
+                                       type: String),
+          FastlaneCore::ConfigItem.new(key: :is_scheduled_release,
```

Comment:
`ConfigItem` provides `default_value` parameter which you could leverage here instead of using  `|| false` in the action body 👍